Common/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# c stands for Common, i stands for Interface
+add_library(ci_libraries_include INTERFACE)
+add_library(ci_libraries_source INTERFACE)
```

Comment:
To be consistent with `gz_` I think this should be named `gzi_` and not `ci_`.

On the other hand, let us say `gz_` is also meant to be used for another game, like BFME, then perhaps these should be named `c_` and `ci_` (or `b_`, `bi_` if we use a "Base" folder).

I think either way is fine, but should be consistent.